src/test/ccl/ut_cclut_mock.inc

```diff
@@ -418,10 +448,202 @@ subroutine test_cclutExecuteProgramWithMocks_mock_subroutine(null)
     call cclutAsserti4Equal(CURREF, "test_cclutExecuteProgramWithMocks_mock_subroutine 005",
         cclut::internalRecord->item, 1)
 end ;test_cclutExecuteProgramWithMocks_mock_subroutine
+declare cclut::mockSubroutine(null) = null with protect
+subroutine cclut::mockSubroutine(null)
+    set mock_subroutine = 1
+end
+
+
+/**
+    Verifies that cclutExecuteProgramWithMocks applies replace option 6 when applying mock table substitutions
```

Comment:
The comments for this test and test_table_replace_9_3 are reversed.  This test is testing for Option 0 on pre-9.3 and test_table_replace_9_3 is testing for Option 6 on 9.3